### PR TITLE
Resolve #84 Condense 2.C.2

### LINKED ISSUE #84: 2.C.2 Can be condensed
Check one:
- [x] Semantic Issue: something about the meaning of the text is different
- [ ] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:
The bulleted list can be condensed to 3 bullet points

### PR DESCRIPTION
**Pull request by esoccoli**
Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Resolve #84

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -122,11 +122,9 @@
 \asubsection{Maintainer Expectations}
 Maintainers are expected to:
 \begin{itemize}
-	\item Review changes to the Constitution for grammar, spelling, and internal consistency
-	\item Keep a public record of changes to the Constitution
```
Comment (rhochgraf21) [inline constitution.tex:126]: I like this requirement/expectation, personally. Candidate for keeping it?

**Review by rhochgraf21 — APPROVED** [review]
I like the idea that Maintainers are expected to keep public record of changes.

**Review by rhochgraf21 — CHANGES_REQUESTED** [review]
whoops this should be

**Comment by esoccoli** [comment]
Made requested changes

**Comment by rhochgraf21** [comment]
👍

**Review by rhochgraf21 — APPROVED** [review]

**Pull request merged**